tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/CelebornFetchFailureSuite.scala

```diff
@@ -86,13 +76,7 @@ class CelebornFetchFailureSuite extends AnyFunSuite
 
   test("celeborn spark integration test - unregister shuffle with throwsFetchFailure disabled") {
```

Comment:
It is special here, please see the UT name,

```
test("celeborn spark integration test - unregister shuffle with throwsFetchFailure disabled")
```

Reply:
The current abstraction of `createSparkSession` is not generic and hard to extend.